examples/http3_client.py

```diff
@@ -129,26 +136,39 @@ def __init__(self, *args, **kwargs) -> None:
         else:
             self._http = H3Connection(self._quic)
 
-    async def get(self, url: str, headers: Dict = {}) -> Deque[H3Event]:
+    async def get(self, url: str, headers: Optional[Dict] = None) -> Deque[H3Event]:
         """
         Perform a GET request.
         """
+        if headers is None:
+            headers = {}
+
         return await self._request(
             HttpRequest(method="GET", url=URL(url), headers=headers)
         )
 
-    async def post(self, url: str, data: bytes, headers: Dict = {}) -> Deque[H3Event]:
+    async def post(
+        self, url: str, data: bytes, headers: Optional[Dict] = None
+    ) -> Deque[H3Event]:
         """
         Perform a POST request.
         """
+        if headers is None:
+            headers = {}
+
         return await self._request(
             HttpRequest(method="POST", url=URL(url), content=data, headers=headers)
         )
 
-    async def websocket(self, url: str, subprotocols: List[str] = []) -> WebSocket:
+    async def websocket(
+        self, url: str, subprotocols: Optional[List[str]] = None
+    ) -> WebSocket:
         """
         Open a WebSocket.
         """
+        if subprotocols is None:
```

Comment:
This doesn't seem necessary, as there is an `if subprotocols` later in this function

Reply:
Fixed.